ras_party/models/models.py

```diff
@@ -318,12 +318,14 @@ def to_respondent_dict(self):
             "telephone": self.telephone,
             "status": RespondentStatus(self.status).name,
             "markForDeletion": self.mark_for_deletion,
-            "associations": self._get_business_associations(self.businesses),
```

Comment:
This is causing a bit of extra processing we don't need. I have done it for this story, but will also be raising a card for all the other to see why and if it is needed